### ISSUE TITLE
Can this work on Heroku?

### ISSUE THREAD

**Issue by juliustip** [PhABC/antiScamBot_slack#18]
A simple instruction on how to push this to Heroku would be appreciated.

**Comment by PhABC** [comment]
I don't think this would work on Heroku, but I am not too sure how Heroku works, so will need to investigate.
